DiscordIntegration.cs

```diff
@@ -0,0 +1,211 @@
+using System;
+using System.Collections;
+using RavenM.DiscordGameSDK;
+using Steamworks;
+using UnityEngine;
+
+namespace RavenM
+{
+    public class DiscordIntegration : MonoBehaviour
+    {
+        public static DiscordIntegration instance;
+        
+        public Discord Discord;
+
+        public long discordClientID = 1007054793220571247;
```

Comment:
This is still the testing app you created, right?

Reply:
Yes, perhaps, for the release, would be appropriate to have your own app instead. I will share you the image resources if you need 'em